software/controller/test/eeprom/test_eeprom.h

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2020-2022, RespiraWorks
```

Comment:
we have "test_eeprom" and we have "eeprom_test". Which is a little bit disorienting. Maybe one of those is a "mock_eeprom"?

Reply:
sure, mock_eeprom.h makes sense